feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,1025 @@
+		af4.Enabled = ygot.Bool(true)
+		rpl := af4.GetOrCreateApplyPolicy()
+		rpl.SetExportPolicy([]string{acceptRoutePolicy})
+		rpl.SetImportPolicy([]string{acceptRoutePolicy})
+
+		af6 := pg.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST)
+		af6.Enabled = ygot.Bool(true)
+		rpl = af6.GetOrCreateApplyPolicy()
+		rpl.SetExportPolicy([]string{acceptRoutePolicy})
+		rpl.SetImportPolicy([]string{acceptRoutePolicy})
+	}
+
+	for _, a := range []*aggPortData{agg1, agg2, agg3} {
+		bgpNbrV4 := bgp.GetOrCreateNeighbor(a.ateIPv4)
+		bgpNbrV4.PeerGroup = ygot.String(pgName)
+		bgpNbrV4.PeerAs = ygot.Uint32(asn)
+		bgpNbrV4.Enabled = ygot.Bool(true)
+		bgpNbrV4T := bgpNbrV4.GetOrCreateTransport()
+
+		bgpNbrV4T.LocalAddress = ygot.String(a.dutIPv4)
+		af4 := bgpNbrV4.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST)
+		af4.Enabled = ygot.Bool(true)
+
+		af6 := bgpNbrV4.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST)
+		af6.Enabled = ygot.Bool(false)
+
+		bgpNbrV6 := bgp.GetOrCreateNeighbor(a.ateIPv6)
+		bgpNbrV6.PeerGroup = ygot.String(pgName)
+		bgpNbrV6.PeerAs = ygot.Uint32(asn)
+		bgpNbrV6.Enabled = ygot.Bool(true)
+		bgpNbrV6T := bgpNbrV6.GetOrCreateTransport()
+
+		bgpNbrV6T.LocalAddress = ygot.String(a.dutIPv6)
+		af4 = bgpNbrV6.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST)
+		af4.Enabled = ygot.Bool(false)
+		af6 = bgpNbrV6.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST)
+		af6.Enabled = ygot.Bool(true)
+	}
+
+	gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Config(), niProto)
+}
+
+// configureRoutingPolicy configure routing policy on DUT
+func configureRoutingPolicy(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
+	d := &oc.Root{}
+	rp := d.GetOrCreateRoutingPolicy()
+	pdef := rp.GetOrCreatePolicyDefinition(acceptRoutePolicy)
+	stmt, _ := pdef.AppendNewStatement("20")
+	stmt.GetOrCreateActions().PolicyResult = oc.RoutingPolicy_PolicyResultType_ACCEPT_ROUTE
+	gnmi.Replace(t, dut, gnmi.OC().RoutingPolicy().PolicyDefinition(acceptRoutePolicy).Config(), pdef)
+}
+
+// configureDUTISIS configure ISIS on DUT
+func configureDUTISIS(t *testing.T, dut *ondatra.DUTDevice, aggIDs []string) {
+	t.Helper()
+	d := &oc.Root{}
+	netInstance := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
+	prot := netInstance.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, isisInstance)
+	prot.Enabled = ygot.Bool(true)
+	isis := prot.GetOrCreateIsis()
+	globalISIS := isis.GetOrCreateGlobal()
+
+	if deviations.ISISInstanceEnabledRequired(dut) {
+		globalISIS.Instance = ygot.String(isisInstance)
+	}
+	globalISIS.LevelCapability = oc.Isis_LevelType_LEVEL_2
+	globalISIS.Net = []string{fmt.Sprintf("%v.%v.00", dutAreaAddress, dutSysID)}
+	globalISIS.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV4, oc.IsisTypes_SAFI_TYPE_UNICAST).Enabled = ygot.Bool(true)
```

Comment:
Default isis use 4 max path to do perform ecmp. Verified the same and it worked.

Reply:
4 paths by default can be true in a specific vendor implementation, but OpenConfig doesn't specify that 
https://github.com/openconfig/public/blob/541fbd64f9a399caed2878aed344c20a93dcbad7/release/models/isis/openconfig-isis.yang#L311

Reply:
@rahulsy26 , as the default value can be different for each vendor when it is not specified explicitly in openconfig requirements so would recommend to set the explicit value value based on the test config  so that it will work for all vendors.

Reply:
I do not understand why setting max ecmp paths is helpful here?

In any case, if the OC default is not being honored, it is ok to configure it, but it should be done using a deviation.

Reply:
Ah, you are correct, there is no default value for isis max-ecmp-paths.     I guess since it is ambiguous what the value is, setting it to some deterministic value (in case implementation is set to `1`)  is ok.  @rahulsy26 if the target device does not support max-ecmp-paths, then a deviation can be added to skip setting max-ecmp-paths.


[/network-instances/network-instance/protocols/protocol/isis/global/config/max-ecmp-paths
](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-isis-global-config-max-ecmp-paths)description: ISIS max-paths count.
nodetype: leaf (rw)
type: uint8

Reply:
Done